core/test/sparse_counts_test.cpp

```diff
@@ -8,8 +8,8 @@ using namespace scipp;
 using namespace scipp::core;
 
 static auto make_sparse() {
-  auto var = makeVariable<double>({Dim::Z, Dim::Y, Dim::X},
-                                  {3, 2, Dimensions::Sparse});
+  auto var = createVariable<double>(Dims{Dim::Z, Dim::Y, Dim::X},
+                                    Shape{3l, 2l, Dimensions::Sparse});
```

Comment:
Didn't we discuss yesterday how the need for using `3l` etc. could be removed?

Reply:
I'm not going redo this stuff arrange the new issue if it annoys you. I takes time to process all the files with refactoring tool, I want to get rid of `makeVariable` as fast as possible, to avoid manual job by merging with other branches in the future.